README.md

```diff
@@ -1,64 +0,0 @@
-## Environment
```

Comment:
For later: For the end of the project, make sure you have a nice project `Readme` that guides reviewers through your repo and gives a quick overview what to find where.